partcad/src/partcad/provider_data_cart.py

```
@@ -59,14 +56,18 @@ def __init__(self):
         self.count = 0
         self.format = "none"
 
+    def _set_store_data(self, part):
```

Ruff (0.8.2)

Missing return type annotation for private function `_set_store_data`

Add return type annotation: `None`

(ANN202)